test/contrib/emscripten/test_emscripten.py

```diff
@@ -0,0 +1,381 @@
+from __future__ import annotations
+
+import typing
+
+import pytest
+
+from urllib3.fields import _TYPE_FIELD_VALUE_TUPLE
+
+# only run these tests if pytest_pyodide is installed
+# so we don't break non-emscripten pytest running
+pytest_pyodide = pytest.importorskip("pytest_pyodide")
+
+from pytest_pyodide import run_in_pyodide  # type: ignore[import] # noqa: E402
+from pytest_pyodide.decorator import (  # type: ignore[import] # noqa: E402
+    copy_files_to_pyodide,
+)
+
+from .conftest import PyodideServerInfo, ServerRunnerInfo  # noqa: E402
+
+# make our ssl certificates work in chrome
+pytest_pyodide.runner.CHROME_FLAGS.append("ignore-certificate-errors")
+
+
+@copy_files_to_pyodide(file_list=[("dist/*.whl", "/tmp")], install_wheels=True)  # type: ignore[misc]
```

Comment:
This decorator is repeated a bunch, probably worth making its own? Also `/tmp` is Linux-specific, might need to use `tempfile`?

Reply:
have done. /tmp is the destination on the emscripten side, so is right.